packages/eas-cli/tsconfig.json

```diff
@@ -14,8 +14,8 @@
     "outDir": "build",
     "rootDir": "src",
     "typeRoots": [
-      "../../node_modules/@types",
       "../../ts-declarations",
```

Comment:
Why do we need to change the order of type directories?

Reply:
Because my type declaration for `prompts/elements/multiselect` is overridden by DefinitelyTyped definition in node_modules (which doesn't define this particular element).